components/notifications/notifications.routes.js

```diff
@@ -0,0 +1,19 @@
+import { Router } from "express";
+import Notification from "./notification.model.js";
+
+const notificationRouter = Router();
+
+notificationRouter.get("/notifications", async (req, res) => {
```

Comment:
also here like u did with conversation , separate the logic from the routes in controller file